Source/WebCore/css/parser/CSSPropertyParserHelpers.cpp

```diff
@@ -6981,14 +6981,21 @@ static RefPtr<CSSPathValue> consumeBasicShapePath(CSSParserTokenRange& args, Opt
             return nullptr;
     }
 
-    if (args.peek().type() != StringToken)
-        return nullptr;
+    auto pathValueFromStringView = [&](const StringView& view) -> RefPtr<CSSPathValue> {
```

Comment:
We should have a guideline about `StringView` and `const StringView&` but I don't think we do 😬

Reply:
Using StringView by value is preferred

Reply:
Will fix in the updated commit.